src/App.js

```diff
@@ -1,24 +1,42 @@
 import React from 'react';
-import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foodsArr from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
+import SearchFood from './components/Search';
 
 function App() {
+  const [foodsList, setFoodsList] = useState(foodsArr);
+
+  function displayFoods() {
+    setFoodsList(foodsList);
+  }
+
+  function addFood(newFood) {
+    setFoodsList((prevFood) => {
+      return [newFood, ...prevFood];
+    });
+  }
+
+  function searchFood(foodToSearch) {
+    setFoodsList((prevFood) => {
+      const newFoodList =
+        foodToSearch === ''
+          ? foodsList
```

Comment:
If you want your whole list to be shown again if searchinput is empty, you have to call your function again, so instead of only 'foodsList' :
```
? setFoodsList(foodsArr);
```